MoreCyclopsUpgrades/Modules/Recharging/Nuclear/NuclearCharger.cs

```diff
@@ -32,6 +32,8 @@ public static void Patch(AssetBundle assetBundle)
 
             // Ensure that the new in-game item is classified as a Cyclops upgrade module. Otherwise you can't equip it.
             CraftDataPatcher.customEquipmentTypes[CyNukBatteryType] = EquipmentType.CyclopsModule;
+
+            DepletedNuclearModule.Patch();
```

Comment:
This call should be moved into the QPatch class

Reply:
Changed